Twitch/Twitch.Base/Services/NewAPI/WebhooksService.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Twitch.Base.Models.NewAPI.Webhooks;
+
+namespace Twitch.Base.Services.NewAPI
+{
+    /// <summary>
+    /// The APIs for Webhook-based services.
+    /// </summary>
+    public class WebhooksService : NewTwitchAPIServiceBase
+    {
+
+        /// <summary>
+        /// Creates an instance of the WebhooksService
+        /// </summary>
+        /// <param name="connection"></param>
+        public WebhooksService(TwitchConnection connection) : base(connection) { }
+
+        /// <summary>
+        /// Gets the Webhook subscriptions of a user identified by a Bearer token, in order of expiration.  Requires App Token.
+        /// </summary>
+        /// <returns></returns>
+        public async Task<IEnumerable<WebhookSubscriptionModel>> GetWebhookSubscriptions(int maxResults=100)
+        {
+            if (maxResults < 10)
```

Comment:
Is there some specific requirement on the results to be between 10 & 100?

Reply:
If so, let's add a Validator.Validate() call to check that it's within the bounds rather than specifically adjusting the maxResults value.

Reply:
There is a min and max for that call.  I will look at the validate

Reply:
Completed